src/fermilib/transforms/_bksf.py

```diff
@@ -0,0 +1,384 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""BKSF transform on fermionic operators."""
+#from __future__ import absolute_import
+
+import numpy as np
+from fermilib.ops import InteractionOperator
+from projectq.ops import QubitOperator
+import networkx as nx
+
+
+def bksf(operator):
```

Comment:
I have changed the names to bravyi_kitaev_fast. I was thinking of bravyi_kitaev_super_fast, but that would've been too long, so I went with your suggestion. But I am planning to still use the name bravyi-kitaev super fast, because in their paper they use the name "super fast algorithm for fermions". Would that be a problem?
Also, even now, some of the method name are really long. So, I am thinking of removing bravyi_kitaev_fast from them. I can use some more suggestions for naming. Let me know.